beacon-chain/sync/payload_attestations.go

```diff
@@ -58,7 +58,7 @@ func (s *Service) validatePayloadAttestation(ctx context.Context, pid peer.ID, m
 		return pubsub.ValidationReject, err
 	}
 
-	if err := v.VerifyBlockRootSeen(s.hasBadBlock); err != nil {
+	if err := v.VerifyBlockRootSeen(s.seenBlockRoot); err != nil {
```

Comment:
Dont you need to move the ordering now? to where invalid is before seen?

Reply:
well, we refactored these helpers so that we keep the same ordering as the spec. Seen already checks for invalid and returns true.

Reply:
But this is wrong, if it's invalid, we want to REJECT, not IGNORE